GameSalesApi/GameSalesApi/Features/AccountManagement/ImageService.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Imgur.API.Authentication.Impl;
+using Imgur.API.Endpoints.Impl;
+using Imgur.API.Models;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Options;
+using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace GameSalesApi.Features.AccountManagement
+{
+    /// <summary>
+    /// A service, responsible for managing user images
+    /// </summary>
+    public class ImageService
```

Comment:
Better rework it on command if you want to use it on POST/DELETE/UPDATE or query if you want to use it with GET